packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -337,20 +337,22 @@ export async function copyProjectTemplateAndReplace(
 
   // Once we are publishing to nuget.org, this shouldn't be needed anymore
   if (options.experimentalNuGetDependency) {
-    [
+    const nugetMappings = [
       {
         from: path.join(sharedPath, projDir, 'NuGet.Config'),
         to: path.join(windowsDir, 'NuGet.Config'),
       },
-    ].forEach(mapping =>
-      copyAndReplaceWithChangedCallback(
+    ];
+
+    for (const mapping of nugetMappings) {
+      await copyAndReplaceWithChangedCallback(
```

Comment:
Should this execute in serial or could `for await (const mapping of nugetMappings) { … }` work too?

Reply:
We want this one to be serial, since the change callbacks present a prompt for each file.

Reply:
Gotcha :+1: